.github/workflows/build-cli-docker.yml

```diff
@@ -0,0 +1,166 @@
+---
+name: Build bws Docker image
+
+on:
+  push:
+    paths:
+      - "crates/bws/**"
+  workflow_call:
+    inputs:
+      sdk_branch:
+        type: string
+        default: master
+      is_workflow_call:
+        type: boolean
+        default: true
```

Comment:
Do you plan to trigger this workflow from another one? If not, this is not needed 😃

```suggestion
```